frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -33,26 +66,220 @@ describe('git import flow', () => {
+    await enterText(resourceLimitsObj.cpuLimit, '0');
+    expect(resourceLimitsObj.cpuRequestHelperText.getText()).toEqual(
+      'CPU request must be less than or equal to limit.',
+    );
+    expect(resourceLimitsObj.cpuLimiHelperText.getText()).toEqual(
+      'CPU limit must be greater than or equal to request.',
+    );
+    await enterText(resourceLimitsObj.memoryRequest, '1');
+    await enterText(resourceLimitsObj.memoryLimit, '0');
+    expect(resourceLimitsObj.memoryRequestHelperText.getText()).toEqual(
+      'Memory request must be less than or equal to limit.',
+    );
+    expect(resourceLimitsObj.memoryLimitHelperText.getText()).toEqual(
+      'Memory limit must be greater than or equal to request.',
+    );
+    await setResources(
+      testData.cpuRequest,
+      testData.cpuLimit,
+      testData.memoryRequest,
+      testData.memoryLimit,
+    );
+  });
+
+  it('add Adanced option "Scaling" details', async () => {
+    // Add Advanced option - Scaling
+    await selectAdvancedOptions(AdvancedOptions.Scaling);
+    expect(await scalingObj.replicaCount.getAttribute('value')).toEqual('1');
+    await setScaling(testData.scalingLimit);
+    expect(await scalingObj.replicaCount.getAttribute('value')).toEqual('2');
+    await scalingObj.increment.click();
+    expect(await scalingObj.replicaCount.getAttribute('value')).toEqual('3');
+    await scalingObj.decrement.click();
+    expect(await scalingObj.replicaCount.getAttribute('value')).toEqual('2');
+  });
+
+  it('add Adanced option "Label" details', async () => {
+    // Add Advanced option - Label
+    await selectAdvancedOptions(AdvancedOptions.Labels);
+    await setLabel(testData.label);
+  });
+
+  // Click on create button
+  it('select create button', async () => {
+    await browser.wait(until.elementToBeClickable(createButton), WAIT);
+    expect(createButton.isEnabled());
+    await createButton.click();
+    await browser.wait(until.urlContains(`${appHost}/topology/ns/${testName}`));
+    expect(browser.getCurrentUrl()).toContain(`${appHost}/topology/ns/${testName}`);
+  });
+
+  /**
+   * Navigate to topology and verify the app presence
+   * Select that app and delete the app
+   * Note: App deletion is targetted in list view, In future, will plan it in graph view
+   */
+  it('verify the app details and delete the component in topology', async () => {
+    await naviagteTo(NavigationMenu.Topology);
+    await click(topologyViewObj.switchToListView);
+    const count = await verifyCreatedAppsInTopology();
+    expect(count).toBeGreaterThanOrEqual(1);
+    await click(listViewObj.appNames.get(0));
+    await selectActionInSideBar(Actions.DeleteDeployment);
+    const actualCount = await listViewObj.appNames.count();
+    expect(actualCount).toBeLessThan(count);
+  });
+});
+
+// This describe block is disabled beacause "Add Pipeline" functionality displays only when pipeline operator is installed
+// So will move this block to pipelinescenarios
```

Comment:
When you want to suggest a future action that you're not taking in this PR, recommend using the TODO standard:

```suggestion
// TODO: Move this block to pipelinescenarios
```

IDEs can pick up on this kind of comment and list out the TODOs for a project.